rs/registry/canister/src/common/test_helpers.rs

```diff
@@ -96,6 +96,7 @@ pub fn get_invariant_compliant_subnet_record(node_ids: Vec<NodeId>) -> SubnetRec
 pub fn prepare_registry_with_nodes(
     start_mutation_id: u8,
     nodes: u64,
+    node_operator_id: Option<PrincipalId>,
```

Comment:
So here, I would make a new function that takes the node_operator_id, and then reimplment the original function in terms of that new function.  That way, everything that doesn't actually care about the node_operator_id (which is most of the existing tests) doesn't need to specify the arg in test setup.